@AH-64D Apache Longbow/Addons/fza_ah64_mpd/pages/ase_draw.hpp

```diff
@@ -429,6 +429,7 @@ class ase_draw {
     };
 };
 
+/*
 class ase_threats_gnd {
     condition = C_COND(C_NOT(C_MPD_USER(MFD_IND_ASE_RLWR_PWR)));
     color[] = {1,1,0,1};
```

Comment:
You would need to leave at least the Msl drawing for the Uk variant unless you also plan to add that the same way

Reply:
We're not mixing the two.

Reply:
So you plan to add this to be drawn aswell then?